build.gradle

```diff
@@ -68,11 +68,36 @@ dependencies {
 	implementation('org.camunda.bpm:camunda-engine-spring:7.9.0')
 	implementation('com.fasterxml.uuid:java-uuid-generator:3.1.2')
 
-	runtimeOnly('org.postgresql:postgresql')
+	implementation('org.postgresql:postgresql')
 	compileOnly group: 'org.projectlombok', name: 'lombok', version: '1.18.6'
 	annotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.18.6'
 
 	testImplementation('org.springframework.boot:spring-boot-starter-test')
 	testImplementation("org.apache.camel:camel-test-spring:${camelVersion}")
 	testImplementation('org.assertj:assertj-core')
+
 }
+
+sourceSets {
+	integrationTest {
+		java.srcDir 'src/integration-test/java'
+		resources.srcDir 'src/integration-test/resources'
+		compileClasspath += main.output + test.output
+		runtimeClasspath += main.output + test.output
+	}
+}
+
+configurations {
+	integrationTestImplementation.extendsFrom testImplementation
+	integrationTestRuntimeOnly.extendsFrom testRuntime
+}
+
+task integrationTest(type: Test) {
+	description = 'Runs integration tests.'
+	group = 'verification'
+	testClassesDirs = sourceSets.integrationTest.output.classesDirs
+	classpath = sourceSets.integrationTest.runtimeClasspath
+	shouldRunAfter test
+}
+
+check.dependsOn integrationTest
```

Comment:
This isn't a showstoppper, but ideally you'd have a new-line here so it plays better with Git.

(If you were to add a line 104, you'd need to modify line 103 to add the line ending, so `git diff` will be slightly strange)

Reply:
I like it, makes sense, good shout.